src/User/UpdateProfile.jsx

```diff
@@ -1,17 +1,93 @@
-import React from "react";
+import React, { useState } from "react";
+import Input from "../Input";
+import { useHistory } from "react-router-dom";
 import {
-  StyledForm,
-  StyledCard
+	StyledForm,
+	StyledCard,
+	StyledCardHeading
 } from "../StyledFormComponents";
+import { updateUserById } from "../api";
 
-const UpdateProfile = () => {
-  return (
-      <StyledForm>
-          <StyledCard>
-              Edit Profile
-        </StyledCard>
-    </StyledForm>
-  );
+const UpdateProfile = ({
+	setUser,
+	id,
```

Comment:
is id not part of the user object? Whats the reason for passing it in seperately